docs/articles/documentation/test-api/switching-between-frames.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: docs
+title: Switching Between Frames
+permalink: /documentation/test-api/switching-between-frames.html
+---
+# Switching Between Frames
+
+At any moment, a TestCafe test has its scope limited to a single frame. If a webpage contains
```

Comment:
Avoid `frame` term. Frames are parts of legacy `<frameset>` element. This term also used by some browser implementers internally. But it never appears in that meaning in any specs and it's not familiar to end users